websites/A/Apple TV/presence.ts

```diff
@@ -0,0 +1,187 @@
+class AppleTV extends Presence {
+  constructor(presenceOptions: PresenceOptions){
+    super(presenceOptions);
+  }
+
+  getVideo(){
+    return document.querySelector("apple-tv-plus-player")
+                    .shadowRoot.querySelector("amp-video-player-internal")
+                    .shadowRoot.querySelector("amp-video-player")
+                    .shadowRoot.querySelector("#apple-music-video-player") as HTMLVideoElement;
+  }
+
+  getTitle(eyebrow = false){
+    if (this.isWatching()){
+      const title = document.querySelector("apple-tv-plus-player")
+                            .shadowRoot.querySelector("amp-video-player-internal")
+                            .shadowRoot.querySelector("div.info__eyebrow")?.textContent;
+
+      if(title || eyebrow) return title;
+      else return document.querySelector("apple-tv-plus-player")
+                          .shadowRoot.querySelector("amp-video-player-internal")
+                          .shadowRoot.querySelector("div.info__title")?.textContent;
+    }
+    const title = document.querySelector("div.product-header__image-logo.clr-primary-text-on-dark > a > h2")?.textContent;
+
+    if (title) return title;
+    else return document.querySelector("img")?.alt;
+  }
+
+  getEpisodeTitle(){
+    return document.querySelector("apple-tv-plus-player")
+                          .shadowRoot.querySelector("amp-video-player-internal")
+                          .shadowRoot.querySelector("div.info__title")?.textContent;
+  }
+
+  isWatching(){
+    return !!document.querySelector("apple-tv-plus-player")
+                    .shadowRoot.querySelector("amp-video-player-internal")
+                    .shadowRoot.querySelector("div.info__title")?.textContent;
```

Comment:
just add type to it instead of using `?`.
how to check type of div or whatever:
```javascript
document.querySelector("apple-tv-plus-player")
                    .shadowRoot.querySelector("amp-video-player-internal")
                    .shadowRoot.querySelector("div.info__title").constructor.name
```

Reply:
I mean I use it as null checker as it's nullable

Reply:
Okay then.